src/components/input/SelectNext.tsx

```diff
@@ -110,26 +110,37 @@ function useShouldDropUp(
 export type SelectProps<T> = PresentationalProps & {
   value: T;
   onChange: (newValue: T) => void;
-  label: ComponentChildren;
+  buttonContent?: ComponentChildren;
   disabled?: boolean;
+
+  /**
+   * Propagated as `id` attributed for the toggle button.
+   * A random one is generated by default
```

Comment:
```suggestion
   * `id` attribute for the toggle button. This is useful to associate a label with the control.
```

A randomly generated ID is not useful from the outside, since the caller has no way to refer to it. If that randomly generated ID is used internally, that's just an implementation detail.

Reply:
Good point.